### ISSUE TITLE
Swyh-rs.apk ?

### ISSUE THREAD

**Issue by MX10-AC2N** [dheijl/swyh-rs#15]
I would have liked to know, if it was possible to create an Android compatible version, I know a good number of users who have been looking for this kind of tools for a long time but under Android ...
When will Swyh-rs.apk ?

**Comment by dheijl** [comment]
I don't think it is feasible right now: Android support has only recently been added to the cpal and fltk-rs libraries, and it would mean a total redesign of the UI. And I don't have any native (NDK) Android development experience...

**Comment by numanair** [comment]
There are existing tools that work on Android but they usually require root for most features. AirMusic works well for me.

**Comment by dheijl** [comment]
Didn't know about AirMusic, features look impressive!

**Comment by MX10-AC2N** [comment]
I think it's possible to adapt swyh-rs but only for Android 10 and newest, because before this release "root" is necessary to access audio... I hope one day swyh-rs.apk...
